back/engines/free/frontend/app/services/frontend/url_service.rb

```diff
@@ -4,7 +4,7 @@ module Frontend
   # The main purpose of this service is to decouple all assumptions the backend
   # makes about the frontend URLs into a single location.
   class UrlService
-    def model_to_url(model_instance, options = {})
+    def model_to_path(model_instance)
```

Comment:
I chose to refactor `def model_to_url` into 2 defs: `def model_to_path` which is now used by `def model_to_url`.

This should mean `def model_to_url` continues to function as previously, whilst we can now use `def model_to_path` to fix this bug (and for any other case where we need to give the FE a path, not a full URL).

I could have created a new `PathService` but chose not to primarily because of the comment in this file:
```
# The main purpose of this service is to decouple all assumptions the backend
# makes about the frontend URLs into a single location.
```

It could also be argued that this refactoring prevents the code duplication involved in creating a new, very similar service.

Reply:
I think it's a great solution.